app/data/mwss_4.json

```diff
@@ -0,0 +1,3804 @@
+{
+    "data_version": "0.0.1",
+    "introduction": {
+        "description": "",
+        "information_to_provide": [
```

Comment:
I've spoken to Chris and Emma and we're ok to go without this for now. One option would have been to use the `introduction.description` element, but that's not currently being rendered (https://github.com/ONSdigital/eq-survey-runner/issues/909).